arithmetization/src/main/java/net/consensys/linea/zktracer/container/module/StatelessModule.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright ConsenSys Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer.container.module;
+
+import net.consensys.linea.zktracer.container.ModuleOperation;
+import net.consensys.linea.zktracer.container.stacked.StackedSet;
+import org.hyperledger.besu.evm.worldstate.WorldView;
+
+/**
+ * A {@link StatelessModule} is a {@link Module} that contains unordered {@link E} where we don't
+ * keep trace of duplicates.
+ */
+public interface StatelessModule<E extends ModuleOperation> extends Module {
+
+  StackedSet<E> operations();
+
+  @Override
+  default void enterTransaction() {
+    operations().enter();
+  }
+
+  @Override
+  default void popTransaction() {
+    operations().pop();
```

Comment:
A stateless class shouldn't change the internal state.

Reply:
no, it does, it deals with a StackSet, cf previous commet. We call them StateLess because they don't interact with Besu state, but computes operations (addition, etc ...)

Reply:
I think you need to review the naming in this case, stateless in a very specific word in Java that means that the class  doesn't change the internal state.

Reply:
yes I don't like the name either, but in this name we refer to the "evm" state

Reply:
done